shared/constants/network.ts

```diff
@@ -1146,10 +1146,12 @@ export const CHAIN_ID_TOKEN_IMAGE_MAP = {
 /**
  * A mapping for networks with enabled profolio landing page to their URLs.
  */
-export const CHAIN_ID_PROFOLIO_LANDING_PAGE_URL_MAP: Record<Hex, string> = {
+export const CHAIN_ID_PROFOLIO_LANDING_PAGE_URL_MAP: Record<string, string> = {
   [CHAIN_IDS.LINEA_MAINNET]:
     'https://portfolio.metamask.io/explore/networks/linea',
   [CHAIN_IDS.SEI]: 'https://portfolio.metamask.io/explore/networks/sei',
+  [MultichainNetworks.SOLANA]:
+    'https://portfolio.metamask.io/explore/networks/solana',
```

Comment:
This link displays a 404 for me

Reply:
Hi David, this is normal, the feature is behind a feature flag that will only be activated when the page goes live.